Source/Provers/SMTLib/SMTLibProcess.cs

```diff
@@ -125,6 +125,20 @@ public bool IsPong(SExpr sx)
       return sx != null && sx.Name == ":name";
     }
 
+    public ProverDiedException HasProverDied()
```

Comment:
The name of this method implies that it returns a boolean.

Reply:
Method name changed to `GetExceptionIfProverDied`